src/commands/scanner/run/dfa.ts

```diff
@@ -43,29 +42,30 @@ export default class Dfa extends ScannerRunCommand {
 			summary: getMessage(BundleName.RunDfa, 'flags.targetSummary'),
 			description: getMessage(BundleName.RunDfa, 'flags.targetDescription'),
 			delimiter: ',',
+			default: '.',
 			multiple: true
 		})(),
 		// END: Flags for targeting files.
 		// BEGIN: Config-overrideable engine flags.
 		'rule-thread-count': Flags.integer({
 			summary: getMessage(BundleName.RunDfa, 'flags.rulethreadcountSummary'),
-			description: getMessage(BundleName.RunDfa, 'flags.rulethreadcountDescription'),
+			default: 4,
 			env: 'SFGE_RULE_THREAD_COUNT'
 		}),
 		'rule-thread-timeout': Flags.integer({
 			summary: getMessage(BundleName.RunDfa, 'flags.rulethreadtimeoutSummary'),
-			description: getMessage(BundleName.RunDfa, 'flags.rulethreadtimeoutDescription'),
+			default: 900000,
```

Comment:
This is the only default remaining. It does indeed match our definition of:
    static final long DEFAULT_RULE_THREAD_TIMEOUT =
            TimeUnit.MILLISECONDS.convert(15, TimeUnit.MINUTES);
but my concern is that this is defined by the engine and so I wouldn't want the CLI overriding this definition with its own. We should keep a single source of truth.

So yeah, we shouldn't set any of these default values in the CLI command config files.